### ISSUE TITLE
Nans in training

### ISSUE THREAD

**Issue by SubhasreeSengupta** [jsleep/wav2mid#3]
Hi Jon,

We are yet facing another issue. We have been able to proceed to the training part. 
But the training loss is going to nan when the training data has "AkpnCGDd" data set in it. On removing it the training proceeds fine. When ran just by itself it runs fine. So we are unable to figure out what is causing this issue, if it is this data set or something else.

Any suggestions would be of great help. Thanks.
